test/minio/1.5.0/ix_values.yaml

```diff
@@ -0,0 +1,9 @@
+image:
+  pullPolicy: IfNotPresent
+  repository: minio/minio
+  tag: RELEASE.2022-01-28T02-28-16Z
+
+# uid=473(minio)
+runAsUser: 473
+# gid=473(minio)
+runAsGroup: 473
```

Comment:
Wonder if we need fsGroup
```
  fsGroup: 473
  fsGroupChangePolicy: "OnRootMismatch"
  ```
  caveat: I don't know anything about kubernetes.
  
  > By default, Kubernetes recursively changes ownership and permissions for the contents of each volume to match the fsGroup specified in a Pod's securityContext when that volume is mounted. 
  
  Maybe this will allow bypassing need to migrate?

Reply:
>OnRootMismatch: Only change permissions and ownership if permission and ownership of root directory does not match with expected permissions of the volume. This could help shorten the time it takes to change ownership and permission of a volume.